lib/helpers.nix

```diff
@@ -386,23 +386,13 @@ with lib; rec {
     };
   };
 
-  globalVal = val:
-    if builtins.isBool val
-    then
-      (
-        if !val
-        then 0
-        else 1
-      )
-    else val;
-
   mkDefaultOpt = {
     type,
     global,
     description ? null,
     example ? null,
     default ? null,
-    value ? v: (globalVal v),
+    value ? v: v,
```

Comment:
there is  `lib.id` that may be clearer

Reply:
This option gets removed in one of the following commits actually.

Reply:
true...... I even read it